src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -92,30 +72,7 @@ static void setup_track_manager_objects()
  */
 static void setup_track_designer_objects()
 {
-    int32_t numObjects = static_cast<int32_t>(object_repository_get_items_count());
-    const ObjectRepositoryItem* items = object_repository_get_items();
     SelectDesignerObjects();
-    for (int32_t i = 0; i < numObjects; i++)
-    {
-        uint8_t* selectionFlags = &_objectSelectionFlags[i];
-        const ObjectRepositoryItem* item = &items[i];
-        if (item->Type == ObjectType::Ride)
-        {
-            *selectionFlags |= ObjectSelectionFlags::Flag6;
-
-            for (auto rideType : item->RideInfo.RideType)
-            {
-                if (rideType != RIDE_TYPE_NULL)
-                {
-                    if (GetRideTypeDescriptor(rideType).HasFlag(RIDE_TYPE_FLAG_SHOW_IN_TRACK_DESIGNER))
```

Comment:
Would it make sense to remove the RIDE_TYPE_FLAG_SHOW_IN_TRACK_DESIGNER flag from any ride types if we are removing this logic here?

Reply:
From our conversation in this PR, I will be reverting my change here, so this question is no longer relevant. Thanks!